### PR TITLE
Get told by Sentry and not the editorial team, of issues

### PR DESCRIPTION
**Pull request by bjfletcher**
We want to be notified before editorial team come knocking on our door with problems.

This PR should ensure that Bug Duty gets notified immediately of any errors with the stream page app.

### CODE REVIEW

**Comment by i-like-robots** [comment]
LGTM 👍

File: server/controllers/stream.js [hunk 1]
```diff
@@ -95,6 +95,10 @@ module.exports = (req, res) => {
 	// Or fail gracefully
 	.catch(err => {
 		logger.error(err);
-		res.sendStatus(err.status || 500);
+		if (err.status === 404) {
```
Comment (matthew-andrews) [inline server/controllers/stream.js]: neat if this works, where does this get set? :)

**Comment by matthew-andrews** [comment]
also LGTM - but would like to understand a bit more how it works

**Comment by bjfletcher** [comment]
This :) https://github.com/Financial-Times/next-stream-page/blob/master/server/controllers/stream.js#L59

**Pull request merged**